Parsers/Quote of the Day.js

```diff
@@ -0,0 +1,40 @@
+/*
+activation_example: !quote
+regex: !quote
+flags: gmi
+order: 100
+stop_processing: true
+active: true
```

Comment:
Per [CONTRIBUTING.md](https://github.com/ServiceNowDevProgram/SlackerBot/blob/main/CONTRIBUTING.md), whitespace is not supported in these fields